ProcessLib/CoupledSolutionsForStaggeredScheme.h

```diff
@@ -76,9 +76,13 @@ struct LocalCoupledSolutions
 
 std::vector<std::vector<double>> getPreviousLocalSolutions(
```

Comment:
Docu missing for both 'previous' and 'current', especially now it is extended by a non-trivial parameter.

Reply:
Added.